app/src/main/res/values-it/strings.xml

```diff
@@ -171,216 +171,205 @@ Halten Sie sich an die Hygiene- und Verhaltensregeln."</string>
 
     <!-- Hinweis beim Eingabefeld für den Authentifizierungs-Code -->
     <string name="auth_code_input_hint">"000-000"</string>
 
     <!-- Meldungen: Info Titel -->
-    <string name="meldungen_additional_info_title2">"Wie soll ich mich verhalten?"</string>
+    <string name="meldungen_additional_info_title2">Cosa devo fare ?</string>
 
     <!-- Meldungen: Text -->
-    <string name="meldungen_additional_info_text3">"Experten vom BAG beraten Sie telefonisch.
-
-Corona-Infoline
-+41 58 463 00 00"</string>
+    <string name="meldungen_additional_info_text3">Gli esperti di XXX ti daranno informazioni per telefono\\\nCoronavirus infoline +39 00 000 0000</string>
 
     <!-- Meldungen: Jetzt anrufen -->
-    <string name="meldungen_call_button_title">"Jetzt anrufen"</string>
+    <string name="meldungen_call_button_title">Chiama ora</string>
 
     <!-- Bezeichnung für Beratungs-Hotline -->
     <string name="meldungen_hotline_label">"Corona-Infoline"</string>
 
     <!-- Label für Anrufen-Button -->
-    <string name="button_call_now">"Jetzt anrufen"</string>
+    <string name="button_call_now">Chiama ora</string>
 
     <!-- Telefon-Nummer für die Hotline -->
-    <string name="tel_hotline">"+41 58 463 00 00"</string>
+    <string name="tel_hotline">+39 00 000 0000</string>
 
     <!-- Meldungen: Text ohne Hotline Info -->
-    <string name="meldungen_additional_info_text3_reduced">"Experten vom BAG beraten dich telefonisch."</string>
-    <string name="meldungen_infected_title">"Infektion gemeldet"</string>
-    <string name="meldungen_infected_text">"Vielen Dank! Nutzer denen du begegnet bist, werden anonym informiert und können sich isolieren."</string>
-    <string name="foreground_service_notification_errors">"App kann zur Zeit nicht richtig funktionieren:"</string>
-    <string name="foreground_service_notification_error_network_sync">"beim Laden der Daten ist ein Fehler aufgetreten"</string>
-    <string name="foreground_service_notification_error_location_permission">"Standort Berechtigung fehlt (benötigt um auf Bluetooth zuzugreifen)"</string>
-    <string name="foreground_service_notification_error_bluetooth_disabled">"Bluetooth ist deaktiviert"</string>
-    <string name="foreground_service_notification_error_battery_optimization">"Akku Optimierung ist aktiviert (verhindert zuverlässiges Scanning im Hintergrund)"</string>
+    <string name="meldungen_infected_title">Infezione segnalata</string>
+    <string name="meldungen_infected_text">Grazie ! Le persone informate in via anonima si potranno isolare interrompendo la catena di contagio.</string>
+    <string name="foreground_service_notification_error_network_sync">"Errore caricando i dati "</string>
+    <string name="foreground_service_notification_error_bluetooth_disabled">Bluetooth è disattivato</string>
+    <string name="foreground_service_notification_error_battery_optimization">L\'ottimizzazione della battaeria è attivata e impedisce la scansione in background</string>
```

Comment:
L\'ottimizzazione della durata della batteria è stata attivata e impedisce una accurata scansione in background